### PR TITLE
feat: storefront guide

### LINKED ISSUE #32: Developers Guide - Developing a Storefront
- [ ] Add and configure Apollo client
- [ ] Build a product listing page
- [ ] Build a product detail page
- [ ] Build navigation menus
- [ ] Add a way to add an item to a cart
- [ ] Build a cart page
- [ ] Implement cart modification
- [ ] Build a checkout page
- [ ] Build an order view page
- [ ] Add ability to log in
- [ ] Add guest checkout functionality
- [ ] Build an account management page

### PR DESCRIPTION
**Pull request by delagroove**
closes #32 

- [x] Add and configure Apollo client
- [x] Build a product listing page
- [x] Build a product detail page
- [x] Build navigation menus
- [x] Add a way to add an item to a cart
- [x] Build a cart page
- [x] Implement cart modification
- [x] Build a checkout page
- [x] Build an order view page
- [x] Add ability to log in
- [x]  Add guest checkout functionality
- [x] Build an account management page (very short at this point)

### CODE REVIEW

File: guides/storefront-guide/storefront-account-management.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#  Storefront UI Development 
+## Build an account management page
+
+### Index
+This article is part of the [Storefront UI Development Guide](./storefront-intro.md).
```
Comment (brent-hoover) [inline guides/storefront-guide/storefront-account-management.md]: @delagroove Is this format required as part of the "guide" format, i.e. should be we be doing this everywhere?

Comment (delagroove) [reply guides/storefront-guide/storefront-account-management.md]: I think it adds a lot of readability and navigation options. Also the idea was to respect the markdown hierarchy. I'll say go for it.
From the guides:

There are four main structural components across all guides:
1. At a glance
2. What you need
3. The body of the guide (over multiple H2 sections)
4. More resources

I'm using 'Overview' as #1

Comment (brent-hoover) [reply guides/storefront-guide/storefront-account-management.md]: Makes a lot of sense. I think once we figure out which pieces should be `guides` and which should be `docs` we should go back and add this to each of the guides.

**Comment by delagroove** [comment]
closed due to duplicate
https://github.com/reactioncommerce/docs/pull/53

**Pull request closed without merging**